backend/pkg/utils/helpers.go

```diff
@@ -49,3 +53,63 @@ func CallPredictionAgentService(jsonData []byte) (analysisResponse models.IssueA
 	}
 	return
 }
+
+func CompareLogs(incomingLogTails []string, currentIssuesLogTails []string) (isNewIssue bool) {
+	logSimilarityThreshold := 0.8
```

Comment:
Write the constant with upper case letter to distinguish it from other variables